Sources/Controllers/2.0/OATargetInfoViewController.mm

```diff
@@ -191,13 +191,15 @@ - (void)sendNearbyImagesRequest:(OARowInfo *)nearbyImagesRowInfo
         if (mapillaryUrl)
             urlString = [urlString stringByAppendingString:[NSString stringWithFormat:@"&osm_mapillary_key=%@", mapillaryUrl]];
     }
-    NSURL *urlObj = [[NSURL alloc] initWithString:urlString];
+    NSURL *urlObj = [[NSURL alloc] initWithString:[[urlString stringByReplacingOccurrencesOfString:@" "  withString:@"_"] stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding]];
     NSURLSession *aSession = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration defaultSessionConfiguration]];
     [[aSession dataTaskWithURL:urlObj completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
         if (((NSHTTPURLResponse *)response).statusCode == 200) {
             if (data) {
                 NSError *error;
-                NSDictionary *jsonDict = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingAllowFragments error:&error];
+                NSString *safeCharsString = [[NSString alloc] initWithData:data encoding:NSISOLatin1StringEncoding];
```

Comment:
I'm not sure about that. But without it, sometimes when JSON with regional characters is loaded, an error occurs during its serialization. Here is problem part of JSON for example: 

Capela_S_Ant�nio_-Maristela_240710_REFON_23.JPG

It's loaded from here:

https://osmand.net/api/cm_place?lat=-23.039064&lon=-47.889183&osm_image=File:Capela_S_Ant%C3%B4nio_-Maristela_240710_REFON_23.JPG